server/webapp/WEB-INF/spring-cruise-remoting-servlet.xml

```diff
@@ -82,6 +88,17 @@
         </property>
     </bean>
 
+    <bean name="recomputeMaterialPipelineMappingsThread" class="org.springframework.scheduling.timer.TimerFactoryBean">
+        <property name="scheduledTimerTasks">
+            <list>
+                <bean class="org.springframework.scheduling.timer.ScheduledTimerTask"
+                      p:delay="10000"
```

Comment:
Making it configurable felt like an overkill. @jyotisingh agreed with me on this :)